README.md

```diff
@@ -1,5 +1,3 @@
-<p align="center"><img src="logo/horizontal.png" alt="neuron" height="150px"></p>
```

Comment:
I wasn't referring to removing this line but the one below, the one that says `# Neuron`. Sorry if I didn't explain myself correctly.